README.md

```diff
@@ -263,19 +263,35 @@ Administrators privileges to the Datadog Agent directories and files.
 Should you wish to add additional elements to the Agent v6 configuration file
 (typically `datadog.yaml`) that are not directly available
 as attributes of the cookbook, you may use the `node['datadog']['extra_config']`
-attribute. This attribute is a hash and will be marshaled into the configuration
+attribute. 
+
+E.g.
+
+```ruby
+default['datadog']['extra_config']['secret_backend_command'] = '/sbin/local-secrets'`
+```
+
+For nested attributes, use object syntax: 
+
+E.g.
```

Comment:
Good addition, I didn't know it was a problem for nested attributes and that we must use the object syntax 👍 
However, what do you think of sticking with the original example and adding the nested attribute example below? I think it would be more natural to read.

Reply:
Yeah if you think reordering the addition makes sense, that's totally fine. I'm not entirely sure how you want to rearrange it though. If you want to just make another commit with your recommendation that is probably easiest.